docs/configuration.md

```diff
@@ -11,7 +11,7 @@ When the module is loaded on start time, the module configuration can be set on
 
 ## Runtime Configuration
 
-You may set certain configuration options at runtime. Setting a configuration at runtime is done using [CONFIG SET](https://redis.io/commands/config-set/) command. Here also, Each configuration must be prefixed with the module name, `redisgears_2.<configuration name>`.
+You may set certain configuration options at runtime. Setting a configuration at runtime is done using [CONFIG SET](https://redis.io/commands/config-set/) command. Here also, each configuration must be prefixed with the module name, `redisgears_2.<configuration name>`.
```

Comment:
Certain configuration options can be configured during runtime., via the[CONFIG SET](https://redis.io/commands/config-set/) command. Each configuration must be prefixed with the module name, `redisgears_2.<configuration name>`.